packages/ui-view/src/View/__tests__/View.test.js

```diff
@@ -104,16 +103,16 @@ describe('<View />', async () => {
     }
 
     const subject = await mount(
-      <View style={{...styleProps}}>
+      <View style={{ ...styleProps }}>
         <h1>Hello!</h1>
       </View>
     )
 
-    const styles = subject.getDOMNode().style
+    const styles = getComputedStyle(subject.getDOMNode())
 
-    expect(styles['backgroundColor']).to.equal('')
-    expect(styles['borderStyle']).to.equal('')
-    expect(styles['opacity']).to.equal('')
+    expect(styles['backgroundColor']).to.equal('rgba(0, 0, 0, 0)')
+    expect(styles['borderStyle']).to.equal('none')
+    expect(styles['opacity']).to.equal('1')
```

Comment:
Good observation and I completely agree.
Although we are running these test in a Node environment I believe in this case it would be fine to test the browser defaults, but nevertheless it is not good practice to do so.
Fixed.